tests/test_field.py

```diff
@@ -177,6 +202,10 @@ def test_optional_redacting_of_missing_values(template_content, data, expected):
             "((warning?? This is a warning\n text after linebreak))",
             "<span class='placeholder-conditional'>&#40;&#40;warning??</span> This is a warning\n text after linebreak&#41;&#41;",  # noqa
         ),
+        (
+            "Unsafe placeholder: ((name::unsafe))",
+            "Unsafe placeholder: <span class='placeholder'>&#40;&#40;name::unsafe&#41;&#41;</span>",
```

Comment:
not a frontend dev, I don't know if we want to extend the css here like we do with conditional placeholders to just highlight the `Placeholder.name` part.

What we currently have for conditional placeholders:

<img width="509" alt="Screenshot 2025-04-16 at 15 24 24" src="https://github.com/user-attachments/assets/6edae69d-d280-48ce-81c5-fa337127f635" />
<img width="492" alt="Screenshot 2025-04-16 at 15 24 37" src="https://github.com/user-attachments/assets/23f52fc5-2662-4974-83f8-cce4721aa1f0" />